Makefile

```diff
@@ -0,0 +1,26 @@
+CURL               ?= curl
+DOCKER_COMPOSE      = docker-compose
+GOLANGCI_VERSION    = v1.21.0
+GOPATH              = $(shell go env GOPATH)
+
+all: lint test
+
+compose:
+	$(DOCKER_COMPOSE) up -d
+
+compose-down:
+	$(DOCKER_COMPOSE) down
+
+$(GOPATH)/bin/golangci-lint:
+	$(CURL) -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin $(GOLANGCI_VERSION)
+
+hooks:
+	script/install-hooks
+
+lint: $(GOPATH)/bin/golangci-lint
+	$(GOPATH)/bin/golangci-lint run ./...
+
+test: compose
```

Comment:
Would be cool to also add a `benchmark` target that only runs the benchmarks without running the tests because that's something that I always get wrong.

Reply:
Hmm, I can't figure out what syntax this is!

According to [the documentation](https://golang.org/cmd/go/#hdr-Testing_flags) this should do it:

`go test -bench="."` but that runs all tests, **including** benchmarks.

Other than that you could go for something like `-run ^Bench` but it's not mentioned in the documentation.

Do you know how to run benchmarks but not all other tests?

Reply:
Ok this seems to do it: `go test ./... -bench=. -run=^a`